Source/DafnyLanguageServer.Test/GutterStatus/SimpleLinearVerificationGutterStatusTester.cs

```diff
@@ -36,7 +36,7 @@ public class SimpleLinearVerificationGutterStatusTester : LinearVerificationGutt
  .  | :}");
   }
 
-  [TestMethod, Timeout(MaxTestExecutionTimeMs)]
+  [TestMethod/*, Timeout(MaxTestExecutionTimeMs)*/]
```

Comment:
Is this intentional? And if so, should the entire timeout be removed?

Reply:
Not intentional. I was trying to debug this failing test and the timeout was preventing me from fixing it. Will fix soon